backends-common/elasticsearch/src/test/java/org/apache/james/backends/es/DockerElasticSearchSingleton.java

```diff
@@ -0,0 +1,34 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.backends.es;
+
+public class DockerElasticSearchSingleton {
+    public static DockerElasticSearch INSTANCE = new DockerElasticSearch();
+
+    static {
+        INSTANCE.start();
+    }
+
+    public static void start() {
+    }
```

Comment:
Is it normal that the start and stop method do nothing ?

Reply:
that’s tricky cause we don’t want to trigger docker start&stop everytime in tests. But that also is the pattern we apply for singleton docker. you can take a look at cassandra and linshare docker singleton.

WDYT?

Reply:
I was not aware of that.
On the master branch i do not see start/stop method on the cassandra singleton. 
It's a little confusing, but if it's a pattern widespread in james i will not complain too much.

Reply:
Agree with Remy IMO we do not need these methods.

Reply:
Okay (https://github.com/linagora/james-project/pull/2341#discussion_r281909119)